src/app/pages/storage/modules/pool-manager/components/pool-manager-wizard/steps/4-log-wizard-step/log-wizard-step.component.ts

```diff
@@ -8,11 +12,20 @@ import { PoolManagerStore } from 'app/pages/storage/modules/pool-manager/store/p
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class LogWizardStepComponent {
+  @Output() goToLastStep = new EventEmitter<void>();
   protected readonly VdevType = VdevType;
+  readonly helptext = helptext;
 
   protected readonly inventory$ = this.store.getInventoryForStep(VdevType.Log);
-
+  protected readonly hasDataVdevs$ = this.store.topology$.pipe(
```

Comment:
I don't think we need to have this check on all the steps.

Reply:
I think we do. Each step has a next button which is not dependent on whether or not there are data vdevs selected. So, user can select next -> next -> next. Or not see the next button enabled when no data vdevs are selected. User shouldn't be able to open other steps when there's an error in a previous step but that's possible right now by just clicking the step title. I'll create a separate ticket for that.

Reply:
It's true that we probably shouldn't allow to go to other steps if first step or data step is not filled in. But there is no use disabling this button on Log step. User either shouldn't be there, or if he's there disabled button won't communicate what the problem is. We should still let him get to review step to see warnings and errors.